docs/conf.py

```diff
@@ -0,0 +1,173 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+sys.path.insert(0, os.path.abspath('.'))  # noqa
+
+import pulp_2to3_migration
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'pulp-2to3-migration'
+copyright = '2019, Pulp Team'
+author = 'Pulp Team'
+
+
+# The short X.Y version
+version = pulp_2to3_migration.__version__
```

Comment:
@fabricio-aguiar, I don't think there is  __version__ defined in pulp_2to3_migration, please add it.
Have you tried to build docs with this config? It's interesting if it doesn't fail with the absent version.

Reply:
@fabricio-aguiar I checked and I can see the version declared in the setup.py, but you also need to declare it again in this file  https://github.com/pulp/pulp-2to3-migration/blob/master/pulp_2to3_migration/__init__.py

See how other projects declare it again like here https://github.com/pulp/pulpcore/blob/master/pulpcore/__init__.py#L1

Reply:
my bad, I haven't tried to build it before. It really failed, changed it and then I successfully built it

Reply:
thanks @goosemania and @bmbouter